docs/src/polynomial.md

```diff
@@ -95,8 +95,12 @@ the usual ways of constructing an element of a ring.
 (R::PolyRing)(c::elem_type(R))
 (R::PolyRing{T})(a::T) where T <: RingElement
 ```
+The third constructor above cannot be used to coerce a polynomial from one ring
+to another; instead use evaluation, for instance: `f(y)` where `y` is the
```

Comment:
But evaluation is *not* the best way. We can mention it, but really should be recommending `map_coefficient` here, ideally with a `jldoctest` usage example.